contrib/nydus-snapshotter/cmd/containerd-nydus-grpc/pkg/command/flags.go

```diff
@@ -46,6 +46,7 @@ type Args struct {
 	MetricsFile          string
 	EnableStargz         bool
 	DisableCacheManager  bool
+	EnableKata           bool
```

Comment:
Do we still need `EnableKata`? for `runc` and `kata`, snapshotter should be able to just return the same mount slice structure IMO.

Reply:
But we can just always return the nydus options. The fuse mount wrapper would ignore it anyway.

Reply:
Or can ignore it. I see that it is not ignored right now.

Reply:
> But we can just always return the nydus options. The fuse mount wrapper would ignore it anyway.

It sounds good, but if so, in runc, it will also depend on nydus-overlayfs and will increase the cost of use, maybe it is a balance